### PR TITLE
longer abstract and corrections

### PR DESCRIPTION
**Pull request by aaronpeikert**

### CODE REVIEW

**Comment by aaronpeikert** [comment]
We could also take the abstract that was intended for the sips?
This one is just very brief.

File: Workshops.qmd [hunk 1]
```diff
@@ -26,12 +26,21 @@ You will learn the basics of the R programming language for data wrangling in th
 ![](img/moritz-ketzer.png){width="100"} ![](img/aaron-peikert.png){width="100"} Monday 11 September, 14:30-16:30
 
 For selected participants who already know the basics of R, there will be the opportunity to learn the basics of the Julia programming language through a facilitated self-paced tutorial.
+Julia is a high-level dynamic programming language designed for numerical and scientific computing, data analysis, machine learning, and more.
+It is easy to learn and use, with a look and feel that is similar to R(/Python/Matlab), but designed with performance in mind.
```
Comment (moritzketzer) [inline Workshops.qmd]:
```suggestion
Julia's user-friendly interface, resembling popular languages like R, Python, and Matlab, makes it easily accessible for learners. Its focus on performance, however, ensures it delivers computational speeds that often rival, or even surpass, those of intricate low-level languages such as C or Fortran.
```

File: Workshops.qmd [hunk 2]
```diff
@@ -26,12 +26,21 @@ You will learn the basics of the R programming language for data wrangling in th
...
+In fact, Julia can be orders of magnitude faster than R, which makes it a great choice for large-scale data analysis and other computationally intensive tasks.
```
Comment (moritzketzer) [inline Workshops.qmd]:
```suggestion
Therefore, Julia is an ideal choice for large-scale data analysis and other computationally intensive tasks, as it can be orders of magnitude faster than R.
```

**Review by moritzketzer — APPROVED** [review]
Finds super! Danke fürs schreiben!

File: Workshops.qmd (hunk 1 above)
Comment (aaronpeikert) [reply Workshops.qmd]:
```suggestion
Julia's user-friendly interface, resembling popular languages like R, Python, and Matlab, makes it easily accessible for learners.
Its focus on performance, however, enables computational speeds that often rival, or even surpass, those of low-level languages such as C or Fortran.
```

File: Workshops.qmd [hunk 3]
```diff
@@ -26,12 +26,21 @@ You will learn the basics of the R programming language for data wrangling in th
 ![](img/moritz-ketzer.png){width="100"} ![](img/aaron-peikert.png){width="100"} Monday 11 September, 14:30-16:30
 
 For selected participants who already know the basics of R, there will be the opportunity to learn the basics of the Julia programming language through a facilitated self-paced tutorial.
+Julia is a high-level dynamic programming language designed for numerical and scientific computing, data analysis, machine learning, and more.
+It is easy to learn and use, with a look and feel that is similar to R(/Python/Matlab), but designed with performance in mind.
+Therefore, Julia is an ideal choice for large-scale data analysis and other computationally intensive tasks, as it can be orders of magnitude faster than R.
```
Comment (Maximilian-Stefan-Ernst) [inline Workshops.qmd]:
```suggestion
Therefore, Julia is an ideal choice for large-scale data analysis and other computationally intensive tasks, as it can be orders of magnitude faster than R or Python.
```

Just because from my experience many people are under the illusion that python is faster than R, so just faster than R might not be a big selling point, but faster than python might be (in reality of course, its same same)

File: Workshops.qmd [hunk 4]
```diff
@@ -26,12 +26,21 @@ You will learn the basics of the R programming language for data wrangling in th
...
+
+In this workshop, we will introduce you to the basics of Julia including the key features of the language, such as its type system, multiple dispatch, and built-in package manager.
```
Comment (Maximilian-Stefan-Ernst) [inline Workshops.qmd]:
```suggestion
In this workshop, we will introduce you to the basics of Julia and the key features setting it apart from other languages.
```
The abstract is already quite long and I think julia first-time users, who this workshop is aimed at, don't know what multiple dispatch (and a type system) is anyways, so for them it's just complicated words :)

File: Workshops.qmd [hunk 5]
```diff
@@ -26,12 +26,21 @@ You will learn the basics of the R programming language for data wrangling in th
...
+We will also compare Julia to R and show you how it can be used in conjunction with R to extend your programming capabilities.
```
Comment (Maximilian-Stefan-Ernst) [inline Workshops.qmd]:
```suggestion
```

I think this may be too ambitious for the limited time and also I am not shure if I would like to do this

File: Workshops.qmd [hunk 6]
```diff
@@ -26,12 +26,21 @@ You will learn the basics of the R programming language for data wrangling in th
...
+
+This workshop is designed for R users who are interested in learning a new programming language or want to improve their skills in data analysis and scientific computing. No prior experience with Julia is required, but a understanding of R is recommended.
```
Comment (Maximilian-Stefan-Ernst) [inline Workshops.qmd:36]: Question: is this the audicence? I understood that there will be an R workshop in parallel, so can only people attend that have prior R knowledge? Or could it be that there are for example python users, who prefer to go to the julia workshop instead of the R workshow? If so, I would rephrase and say prior programming experience is recommended, and not specifically R experience

File: Workshops.qmd [hunk 7]
```diff
@@ -26,12 +26,21 @@ You will learn the basics of the R programming language for data wrangling in th
 ![](img/moritz-ketzer.png){width="100"} ![](img/aaron-peikert.png){width="100"} Monday 11 September, 14:30-16:30
 
 For selected participants who already know the basics of R, there will be the opportunity to learn the basics of the Julia programming language through a facilitated self-paced tutorial.
```
Comment (Maximilian-Stefan-Ernst) [inline Workshops.qmd:28]: Only for R-users or also for experienced Fortran-Programmers preferring to learn julia over R? ;) (see my comment below)

Comment (aaronpeikert) [reply Workshops.qmd:28]: I haven't written that part. Malika did. I think most of the other workshops require knowledge of R.

File: Workshops.qmd (hunk 6 above)
Comment (aaronpeikert) [reply Workshops.qmd:36]: No it will most likely all people who know R. The problem ist, that most other workshops require R. So it does not make sense to prefer the Julia workshop if you do not know R if you want to attend the whole summer school. However, some people may only want to attend the Julia workshop. See https://github.com/MalikaIhle/OSC-Open-Research-Summer-School-2023/issues/2

**Comment by aaronpeikert** [comment]
@moritzketzer @Maximilian-Stefan-Ernst If you want other changes feel free to open another PR, but I would now make a PR against upstream.
